data-transform/src/main/java/com/linkedin/data/transform/filter/MaskComposition.java

```diff
@@ -90,6 +90,11 @@ public void interpret(InterpreterContext instrCtx)
                          instrCtx);
           }
         }
+        // This can happen if the mask is for an array field and the merged start/count resulted in default values.
+        // Setting the wildcard mask to represent all items are included.
+        if (data.isEmpty()) {
```

Comment:
although i like this coding style, but it seems it is not consistent with rest of the code.

Reply:
I mean the left bracket should be in a new line

Reply:
Fixed.